kratos/gui/screen.py

```diff
@@ -0,0 +1,49 @@
+"""
+The base class for creating more complex GUI screens
+"""
+
+import os
+
+import PyQt5.QtCore as QtCore
+import PyQt5.QtGui as QtGui
+import PyQt5.QtQuick as QtQuick
+
+
+class Screen:
+    """
+    Base screen class that creates an Qt application with argument passing and QML
+    resource loading
+    """
+
+    # pylint: disable=too-few-public-methods
+    # This is a base class and has (at the moment) only limited functionality
+
+    _argv = []
```

Comment:
Should this be a parameter to constructor instead of an attribute?

Reply:
Yeah, a parameter to the ctor is probably less confusing.